src/api/guides.client.ts

```diff
@@ -0,0 +1,17 @@
+import { GuideResponse } from "../app/(admin)/new_dashboard/guides/types";
+import { BaseClient } from "./base.client";
+import axios from "axios";
+
+export class GuidesClient extends BaseClient {
+  async createGuide<T>(guideData: any) {
+    return this.post<T>("guides", guideData);
+  }
+
+  async getGuides() {
+    return this.get<GuideResponse[]>("guides");
+  }
+
+  async getGuideById(guideId: string) {
+    return this.get<GuideResponse>(`guides/${guideId}`);
+  }
+}
```

Comment:
Why createGuide is generic type of T. Please use specific types, I'd recommend trying to create types with zod.
Also please do not use guideData as any.

Delete the axios import since you don't use it here.